test-utils/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-openapi-generator-parent</artifactId>
+        <groupId>io.quarkiverse.openapi.generator</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-openapi-generator-test-utils</artifactId>
+    <name>Quarkus - Openapi Generator - Test Utils</name>
+
+    <properties>
+        <version.org.assertj>3.22.0</version.org.assertj>
+        <version.com.github.javaparser>3.24.0</version.com.github.javaparser>
```

Comment:
Do we need these versions here? Also, our bot just updated the parser to 3.24.2.

Reply:
Good catch.
I removed them.